### PR TITLE
Add speakers

### PR DESCRIPTION
**Pull request by anssiko**
@dontcallmedom @cvan time to publish the speakers! Any comments before we go live?

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -316,7 +441,11 @@ <h4 id="schedule-day-1">
               <dd>
                 Because VR is much better experienced than talked about, we
                 encourage anyone with demos worth sharing to come prepared to
-                show them
+                show them. In order to facilitate identification of what demos
+                will be available from whom, we encourage you to document your
+                demos in <a href=
+                "https://docs.google.com/document/d/1opAAH6CZhxx5F6c9bvISphHF58ZLE7dS4mJKQDQKRDQ/">
```
Comment (dontcallmedom) [inline schedule.html]: I'm hesitant to put this in the schedule; although that URL is now de facto public, I think I would prefer keeping it to direct messages to participants.

File: schedule.html [hunk 2]
```diff
@@ -413,6 +589,18 @@ <h4 id="schedule-day-2">
...
             </summary>
+            <dl>
+              <dt>
+                Format
+              </dt>
+              <dd>
+                Open one-hour session for discussions in smaller groups on
+                specific topics proposed by workshop participants. Please add
+                your suggestion to <a href=
+                "https://docs.google.com/document/d/1HxnxffzFvbigg_xH4180ROKH1eNNkgLWlCpw_FhH1uQ/">
```
Comment (dontcallmedom) [inline schedule.html]: as above

**Comment by anssiko** [comment]
@dontcallmedom Addressed your comments, good to merge?

File: schedule.html [hunk 3]
```diff
@@ -452,6 +651,16 @@ <h4 id="schedule-day-2">
...
               <dt>
+                Speakers
+              </dt>
+              <dd>
+                Dominique Hazael-Massieux, W3C
+              </dd>
+              <dd>
+                Neil Trevett, Khronos
+              </dd>
+              <dd></dd>
```
Comment (cvan) [inline schedule.html:662]: why any extra?

Comment (anssiko) [reply schedule.html:662]: Fixed.

**Pull request merged**